.gitignore

```diff
@@ -25,6 +25,8 @@ yarn-debug.log*
 yarn-error.log*
 
 .env
+package-lock.json
```

Comment:
This should not be here. We're using `yarn` to install dependencies, no `package-lock.json` file should be in the project.